src/game/game.js

```diff
@@ -413,7 +413,7 @@
             });
 
             if(!runCodeCache[userId].globals._) {
-                runCodeCache[userId].globals._ = _.runInContext();
+                runCodeCache[userId].globals._ = _.runInContext(runCodeCache[userId].globals);
```

Comment:
Exactly, and doing so (parsing lodash code in each context separately) creates huge runtime load according to our tests in the past.